examples/Alert.sticky.jsx

```diff
@@ -4,10 +4,11 @@
  *--------------------------------------------------------------------------------------------*/
 import * as React from 'react';
 import { Alert, Text } from '@itwin/itwinui-react';
+import './Alert.sticky.css';
```

Comment:
We should not be importing CSS files into each component like this. This CSS is considered "global" (unscoped) and will cause unnecessary conflicts with other global CSS while forcing us to use long, unwieldy class names.

Reply:
Okay, I've approved the PR set the base to the shadow-demos branch, and merged the changes from that branch in.

Reply:
Alright, will do.

Reply:
Most recent commit has simplified the class names and added the `demo-` prefix to them.